packages/site-content/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "rootDir": "templates/src",
+    "noEmit": true,
+    "tsBuildInfoFile": "tsconfig.tsbuildinfo",
+    "module": "esnext",
+    "moduleResolution": "node",
+},
+  "include": ["templates/src/**/*.ts"],
```

Comment:
I guess disregard my previous comment, but... why isn't esbuild using the `include` setting here?

Reply:
I don't see a way to make esbuild pay attention to `include`. It always requires that you pass it the entrypoints explicitly, as far as I can tell.